more/free-programming-cheatsheets.md

```diff
@@ -263,6 +264,13 @@
 * [TypeScript Next.js Cheatsheet](https://www.saltycrane.com/cheat-sheets/typescript/next.js/latest/) - SaltyCrane Cheat Sheets
 
 
+#### Node.js
+
+* [Node.js Cheatsheet](https://www.codecademy.com/learn/learn-node-js/modules/intro-to-node-js/cheatsheet) - Codecademy (HTML)
+* [Node.js Cheatsheet ZTM](https://zerotomastery.io/cheatsheets/node-js-cheat-sheet) - ZeroToMastery (HTML)
+* [Node.js/Express Cheatsheet](https://courses.cs.washington.edu/courses/cse154/19su/resources/assets/cheatsheets/node-cheatsheet.pdf) - Washington CSE 154: Web Programming (PDF)
```

Comment:
```suggestion
* [Node.js/Express Cheatsheet](https://courses.cs.washington.edu/courses/cse154/19su/resources/assets/cheatsheets/node-cheatsheet.pdf) - Melissa Hovik (PDF)
```
This is the course instructor. Please check the context to see if somebody else should be credited.